docs/micro-rdk/servo/gpio.md

```diff
@@ -0,0 +1,133 @@
+
+{{< tabs name="Configure a gpio Servo" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `servo` type, then select the `gpio` model.
+Enter a name for your servo and click **Create**.
+
+![An example configuration for a gpio servo in the Viam app Config Builder.](/components/servo/gpio-servo-ui-config.png)
+
+Copy and paste the following attribute template into your servo's **Attributes** box.
+Then remove and fill in the attributes as applicable to your servo, according to the table below.
+
+{{< tabs >}}
+{{% tab name="Attributes template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pin": "<your-pin-number>",
+  "board": "<your-board-name>",
+  "frequency_hz": <int>,
+  "min_angle_deg": <int>,
+  "max_angle_deg": <int>,
+  "min_period_us": <int>,
+  "max_period_us": <int>,
+  "pwm_resolution": <int>
+}
+```
+
+{{% /tab %}}
+{{% tab name="Attributes example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pin": "16",
+  "board": "local"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-servo-name>",
+      "model": "gpio",
+      "type": "servo",
+      "namespace": "rdk",
+      "attributes": {
+        "pin": "<your-pin-number>",
+        "board": "<your-board-name>",
+        "frequency_hz": <int>,
+        "min_angle_deg": <int>,
+        "max_angle_deg": <int>,
+        "min_period_us": <int>,
+        "max_period_us": <int>,
+        "pwm_resolution": <int>
+      }
+    }
+  }
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "esp32",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": [16]
+      }
+    },
+    {
+      "name": "my_servo",
+      "model": "gpio",
+      "type": "servo",
+      "namespace": "rdk",
+      "attributes": {
+        "pin": "16",
+        "board": "local"
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `gpio` servos:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `pin` | string | **Required** | The GPIO number of the pin the servo's control wire is wired to on the board. |
+| `board` | string | **Required** | `name` of the [board](/micro-rdk/board/) the servo is wired to. |
+| `frequency_hz` | int | Optional | The frequency of [PMW](https://docs.arduino.cc/learn/microcontrollers/analog-output) pulses sent to the servo, in Hertz (*Hz*). <br> Default = [`300`] <br> Range = [`0`, `450`] |
```

Comment:
just FYI, there's a bunch of places where you have "PMW" rather than "PWM" (like here)

Reply:
Oh no! That's not good lol, it must be mistyped in the regular docs too. Will fix-- thanks so much for catching!